package.json

```diff
@@ -32,24 +32,25 @@
     "url": "http://blog.millermedeiros.com/"
   },
   "devDependencies": {
-    "esprima": "1.0",
-    "mocha": "https://github.com/millermedeiros/mocha/tarball/latest",
     "chai": "1.4",
+    "esprima": "1.0",
     "glob": "3.1",
     "istanbul": "~0.1.23",
     "jshint": "~2.3.0",
+    "mocha": "https://github.com/millermedeiros/mocha/tarball/latest",
     "mockery": "^1.4.0"
   },
   "dependencies": {
+    "debug": "^0.7.4",
     "mout": ">=0.9 <2.0",
+    "npm-run": "https://github.com/millermedeiros/npm-run/tarball/tmp-run-sync",
```

Comment:
yes, it is. I'm waiting for https://github.com/timoxley/npm-run/pull/3 to be merged and published. If it doesn't happen soon I'll probably just publish as a separate package.